creator-node/src/monitors/syncHistory.js

```diff
@@ -1,65 +1,52 @@
-const redisClient = require('../redis')
 const SyncHistoryAggregator = require('../snapbackSM/syncHistoryAggregator')
 
-const get30DayRollingSyncSuccessCount = async () => {
+/**
+ * Get the total number of syncs that met 'status' per wallet per day for the given number of days
+ * @param {String} status Either 'success' or 'fail'
+ * @param {Number} days number of historical days to get data for
+ * @returns {Number} number of syncs per wallet per day that met the status
+ */
+const _getRollingSyncCount = async (status, days) => {
+  if (status !== 'success' && status !== 'fail') throw new Error('Invalid status for _getRollingSyncCount: must be "success" or "fail"')
+
   // Get the start and end dates of the rolling window
   const today = new Date()
   const rollingWindowStartDate = new Date(
-    today.setDate(today.getDate() - 30)
+    today.setDate(today.getDate() - days)
   )
   const rollingWindowEndDate = new Date()
 
-  // Get the success count from the date within the rolling window range
-  let rollingSyncSuccessCount = 0
+  // Get the count for this status from the date within the rolling window range
+  let rollingSyncStatusCount = 0
   let date = rollingWindowStartDate
 
   while (date <= rollingWindowEndDate) { // eslint-disable-line no-unmodified-loop-condition
     const redisDateKeySuffix = date.toISOString().split('T')[0] // ex.: "2021-05-04"
-    const { success: syncSuccessCountKey } = SyncHistoryAggregator.getAggregateSyncKeys(redisDateKeySuffix)
-
-    const syncSuccessCount = await getSyncCount(syncSuccessCountKey)
-    rollingSyncSuccessCount += syncSuccessCount
+    const resp = await SyncHistoryAggregator.getDailyWalletSyncData(redisDateKeySuffix)
+    rollingSyncStatusCount += parseInt(resp[status], 10) || 0
 
     // Set the date to the next day
     date.setDate(date.getDate() + 1)
   }
 
-  return rollingSyncSuccessCount
+  return rollingSyncStatusCount
 }
 
-const get30DayRollingSyncFailCount = async () => {
-  // Get the start and end dates of the rolling window
-  const today = new Date()
-  const rollingWindowStartDate = new Date(
-    today.setDate(today.getDate() - 30)
-  )
-  const rollingWindowEndDate = new Date()
-
-  // Get the fail count from the date within the rolling window range
-  let rollingSyncFailCount = 0
-  let date = rollingWindowStartDate
-
-  while (date <= rollingWindowEndDate) { // eslint-disable-line no-unmodified-loop-condition
-    const redisDateKeySuffix = date.toISOString().split('T')[0] // ex.: "2021-05-04"
-    const { fail: syncFailCountKey } = SyncHistoryAggregator.getAggregateSyncKeys(redisDateKeySuffix)
-
-    const syncFailCount = await getSyncCount(syncFailCountKey)
-    rollingSyncFailCount += syncFailCount
-
-    // Set the date to the next day
-    date.setDate(date.getDate() + 1)
-  }
+const get30DayRollingSyncSuccessCount = async () => {
+  return _getRollingSyncCount('success', 30)
```

Comment:
also update the sync state here **https://github.com/AudiusProject/audius-protocol/pull/1654/files#r670578406**

Reply:
done